core-bundle/contao/templates/twig/content_element/_base.html.twig

```diff
@@ -27,3 +27,6 @@
 {% if searchable is defined and not searchable -%}
 <!-- indexer::continue -->
 {% endif %}
+
+{# Add metadata #}
+{% block schema_org %}{% endblock %}
```

Comment:
Just to be sure I fully understand: The idea of this block is, that someone could disable adding metadata by overwriting it with an empty block again, right? I mean, nothing is ever output, so it could also use the same logic as with `searchable`, i.e. if a variable is defined and true/false, then do/don't do the *thing*, otherwise do the default.

Just thinking out load and not saying that I would change anything. A block is maybe more of an "API" or contract then a variable. :thinking:

Reply:
Well technically - at least in my view - the `{% do %}` tag in a child template should always be allowed outside blocks and then probably executed in the `display_start` section. This would be helpful for other things like `{% set %}` for attributes as well. :thinking: Maybe we should pitch this to the Twig core (or make it happen in a node visitor).

For the short term, I agree, a block makes sense. Should we maybe name it a bit more general then? 
`{% block metadata %}`? Wdyt?

Reply:
Yes, I agree but I had to work with what I have 😅 Renamed the block for now.